website/content/en/docs/component-guides/pipelines.md

```diff
@@ -63,129 +63,51 @@ client = kfp.Client(host=f"https://{kubeflow_gateway_endpoint}/pipeline", cookie
-
-   cat <<EOF > trust.json
-   {
-   "Version": "2012-10-17",
-   "Statement": [
-       {
-       "Effect": "Allow",
-       "Principal": {
-           "Federated": "arn:aws:iam::${AWS_ACCOUNT_ID}:oidc-provider/${OIDC_URL}"
-       },
-       "Action": "sts:AssumeRoleWithWebIdentity",
-       "Condition": {
-           "StringEquals": {
-           "${OIDC_URL}:aud": "sts.amazonaws.com"
-           }
-       }
-       }
-   ]
-   }
-   EOF
-   ```
+## AWS Access from Pipeline Components
+
+For pipelines component to be granted access to AWS resources, the corresponding profile in which the pipeline is created needs to be configured with the `AwsIamForServiceAccount` plugin.
+
+Below is an example of a profile using the `AwsIamForServiceAccount` plugin:
+```yaml
+apiVersion: kubeflow.org/v1
+kind: Profile
+metadata:
+  name: some_profile
+spec:
+  owner:
+    kind: User
+    name: some-user@kubeflow.org
+  plugins:
+  - kind: AwsIamForServiceAccount
+    spec:
+      awsIamRole: arn:aws:iam::123456789012:role/some-profile-role
+```
 
-6. [Create an IAM policy](https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_create.html) to scope the permissions for the Profile. For simplicity, we will use the `arn:aws:iam::aws:policy/AmazonS3FullAccess` policy as an example.
+The AWS IAM permissions granted to the pipelines components are specified in the profile's `awsIamRole`. 
 
-7. [Create an IAM role](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_create.html) for the Profile using the scoped policy from the previous step.
+To configure the `AwsIamForServiceAccount` plugin to work with profiles, follow the [Configuration Steps](#configuration-steps) below.
 
-   ```bash
-   aws iam create-role --role-name $PROFILE_NAME-$CLUSTER_NAME-role --assume-role-policy-document file://trust.json
+### Configuration steps
 
-   aws iam attach-role-policy --role-name $PROFILE_NAME-$CLUSTER_NAME-role --policy-arn arn:aws:iam::aws:policy/AmazonS3FullAccess
-   ```
-
-8. Create a Profile using the `PROFILE_NAME`.
+Configuration steps to configure profiles with AWS IAM permissions can be found [here](./profiles.md#configuration-steps).
+The configuration steps will configure the profile controller to work with the `AwsIamForServiceAccount` plugin.
 
-   ```bash
-   cat <<EOF > profile_iam.yaml
-   apiVersion: kubeflow.org/v1
-   kind: Profile
-   metadata:
-     name: ${PROFILE_NAME}
-   spec:
-     owner:
-       kind: User
-       name: user@example.com
-     plugins:
-     - kind: AwsIamForServiceAccount
-       spec:
-         awsIamRole: $(aws iam get-role --role-name $PROFILE_NAME-$CLUSTER_NAME-role --output text --query 'Role.Arn')
-   EOF
-
-   kubectl apply -f profile_iam.yaml
-   ```
+### Example: S3 Access from a Pipeline Component
 
-## Verification steps
+The below steps walk through creating a pipeline with a component that has permissions to list buckets in S3.
+#### Prerequisites
```

Comment:
can we remove Kubeflow installation here because it is required for any section in a component guide? also vanilla can be misleading

Maybe rename the Configuration steps section above -> Prerequisites instead

Reply:
> Maybe rename the Configuration steps section above -> Prerequisites instead

these are prereqs for the example